### ISSUE TITLE
Switch starlark to tree-sitter-starlark?

### ISSUE THREAD

**Issue by auscompgeek** [cursorless-dev/vscode-parse-tree#90]
Back in #44 we added Starlark support, reusing tree-sitter-python since Starlark's grammar is a subset of Python 2's.

Since then, https://github.com/tree-sitter-grammars/tree-sitter-starlark was created. It might be a good idea to switch to using it instead?

**Comment by pokey** [comment]
Definitely, tho we may no longer get it for free, depending whether the parse tree differs significantly from tree-sitter-python
